### PR TITLE
I18n sync Down & Out 12/14

### PR DESCRIPTION
**Pull request by deploy-code-org**
# I18n Sync Down & Out

This PR contains all changes to translations made since the last sync (usually a week).

## To Review:

Look through the changes in each of the commits and verify that all changes are expected. Note that **it is not necessary to review every single "dashboard i18n update" commit.** These commits are batched by language to make it easier to review just a couple commits in this category.

### Things to Note:

- We expect translation changes to come in on a regular basis; new strings will be translated, existing translations will be updated, and source string removals will result in translations for those strings also being removed.

- We expect translations to arrive in large batches, so don't worry if you see a large group of changes in a specific language.

- We _do not_ usually expect to see a large number of formatting changes, or batches of changes that apply to strings in every single language. If you see this, it usually means either that a code change went in or that something broke.

## Known Issues:

- If the CI unit tests fail at the `grunt messages:all` step, they will typically fail with a message like:

  ```
  [2019-10-04 02:38:57] Running "messages:all" (messages) task
  [2019-10-04 02:39:00] Warning: Error processing localization file i18n/common/zh_tw.json: SyntaxError: Expected [0-9a-zA-Z$_] but "\u8A0A" found. Use --force to continue. 
  ```

  This usually means that a translator has translated one of the [MessageFormat variables](https://messageformat.github.io/messageformat/guide/#variables). In this case, you should find and fix the offending translation in crowdin, and then manually update the associated strings in `i18n/locales` and `apps/i18n`.

  See https://github.com/code-dot-org/code-dot-org/pull/31098/commits/ab3e5d1d740869a880f2e9da5cf0a4348886915e and https://github.com/code-dot-org/code-dot-org/pull/31098/commits/0fca37658cacaaff55d9adfc9d04c40b37cc5e2d for an example.

## To Deploy:

Once one or two people have reviewed and approved this change and the tests are passing, ship it!

It's not necessary to wait for everyone on the team to review.

### CODE REVIEW

**Comment by deploy-code-org** [comment]
This comment is created by running: `bin/i18n/analyze_pr.rb -p 49501 -c`

Top 5/701 files sorted by the number of changes:
- `bin/i18n/crowdin/codeorg_etags.json`, 80080 changes (40040 additions, 40040 deletions).
- `dashboard/config/locales/activity_sections.tr-TR.json`, 2423 changes (1267 additions, 1156 deletions).
- `dashboard/config/locales/activity_sections.ca-ES.json`, 1732 changes (1571 additions, 161 deletions).
- `dashboard/config/locales/start_html.ru-RU.json`, 1693 changes (1693 additions, 0 deletions).
- `dashboard/config/locales/start_html.tr-TR.json`, 1660 changes (1660 additions, 0 deletions).

Top 5/90 commits sorted by the number of changes:
- Commit: etags updates
  - 80612 changes (40306 additions, 40306 deletions).
  - 4 files changed.
  - File with the most changes: `bin/i18n/crowdin/codeorg_etags.json`, 80080 changes (40040 additions, 40040 deletions).
- Commit: dashboard i18n updates - Turkish
  - 9585 changes (5902 additions, 3683 deletions).
  - 32 files changed.
  - File with the most changes: `dashboard/config/locales/activity_sections.tr-TR.json`, 2423 changes (1267 additions, 1156 deletions).
- Commit: dashboard i18n updates - French
  - 8058 changes (5155 additions, 2903 deletions).
  - 39 files changed.
  - File with the most changes: `dashboard/config/locales/start_html.fr-FR.json`, 1654 changes (1654 additions, 0 deletions).
- Commit: course content i18n sync
  - 5894 changes (5479 additions, 415 deletions).
  - 27 files changed.
  - File with the most changes: `i18n/locales/source/course_content/other/csd6-2022.json`, 1171 changes (1064 additions, 107 deletions).
- Commit: dashboard i18n updates - Catalan
  - 5620 changes (5389 additions, 231 deletions).
  - 22 files changed.
  - File with the most changes: `dashboard/config/locales/activity_sections.ca-ES.json`, 1732 changes (1571 additions, 161 deletions).

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_create_activities.tr-TR.md.partial [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 
-{{ featured_tutorial, title: "Sanatçı", body: "Sanatçı ile harika resimler ve tasarımlar çizin!", image: "/shared/images/courses/logo_tall_artist.jpg", url: "https://studio.code.org/s/artist/reset" }}
+{{featured_tutorial, title: “Sanatçı”, body: “Sanatçı ile harika resimler ve tasarımlar çizin!”, image: "/shared/images/courses/logo_tall_artist.jpg", url: "https://studio.code.org/s/artist/reset" }}
 
-{{ featured_tutorial, title: "Infinity Oyun Laboratuvarı", body: "Disney Infinity karakterlerinin oynadığı bir hikaye veya oyun oluşturmak için Oyun Laboratuvarını kullanın.", image: "/shared/images/courses/logo_tall_infinity.jpg", url: "https://studio.code.org/s/infinity/reset" }}
+{{ featured_tutorial, title: “Infinity Oyun Laboratuvarı”, body: “Disney Infinity karakterlerinin oynadığı bir hikaye veya oyun oluşturmak için Oyun Laboratuvarını kullanın.", image: "/shared/images/courses/logo_tall_infinity.jpg", url: "https://studio.code.org/s/infinity/reset" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_create_activities.tr-TR.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_create_activities.ar-SA.md.partial [hunk 2]
```diff
@@ -1,6 +1,6 @@
-{{ featured_tutorial, title: "حفلة رقص", body: "قم ببرمجة حفلة رقص! يضم بيونسيه وهاري ستايلز وليزو وليل ناس إكس وسيلينا جوميز وموسيقى من ديزني \"إنكانتو\" والمزيد!", image: "/images/hoc2022-dance-party.png", url: "https://code.org/dance" }}
+{{ featured_tutorial, title: "حفلة رقص"، body: "قم ببرمجة حفلة رقص! يضم الحفل بيونسيه وهاري ستايلز وليزو وليل ناس إكس وسيلينا جوميز وموسيقى من ديزني \"إنكانتو\" والمزيد!", image: "/images/hoc2022-dance-party.png", url: "https://code.org/dance" }}
 
-{{ featured_tutorial, title: "فن القصيدة", body: "أضف الفن والرسوم المتحركة إلى قصيدة كلاسيكية في هذا النشاط من أنشطة ساعة البرمجة!", image: "/shared/images/courses/logo_tall_poem-art.jpg", url: "https://studio.code.org/s/poem-art-2021/reset" }}
+{{ featured_tutorial, title: "فن القصيدة"، body: "أضف الفن والرسوم المتحركة إلى قصيدة كلاسيكية في هذا النشاط من أنشطة ساعة البرمجة!", image: "/shared/images/courses/logo_tall_poem-art.jpg", url: "https://studio.code.org/s/poem-art-2021/reset" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_create_activities.ar-SA.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_explore_activities.ar-SA.md.partial [hunk 3]
```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "مرحبًا أيها العالم: الفضاء", body: "أنشيء مشاهد بين المجرات مع شخصيات وأصوات تقول ، \"مرحبًا بالعوالم!\"", image: "/images/hoc2022-hello-world-space.png", url: "https://studio.code.org/s/hello-world-space-2022/reset" }}
+{{ featured_tutorial, title: "مرحبًا أيها العالم: الفضاء"، body: "أنشيء مشاهد بين المجرات مع شخصيات وأصوات تقول، \"مرحبًا بالعوالم!\"", image: "/images/hoc2022-hello-world-space.png", url: "https://studio.code.org/s/hello-world-space-2022/reset" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_explore_activities.ar-SA.md.partial:1]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_explore_activities.ta-IN.md.partial [hunk 4]
```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "வணக்கம் உலகம்: விண்வெளி", body: "என்று நீங்கள் சொல்லும் எழுத்துக்கள் மற்றும் ஒலிகளைக் கொண்ட இண்டர்கலெக்டிக் காட்சிகளை உருவாக்கவும்.\ஹலோ வேர்ல்ட்ஸ்!\"", image: "/images/hoc2022-hello-world-space.png", url: "https://studio.codeorg/s/hello-world-space-2022/reset" }}
+{{ featured_tutorial, title:"வணக்கம் உலகம்: விண்வெளி", body: "என்று நீங்கள் சொல்லும் எழுத்துக்கள் மற்றும் ஒலிகளைக் கொண்ட இண்டர்கலெக்டிக் காட்சிகளை உருவாக்கவும்.\ஹலோ வேர்ல்ட்ஸ்!\"", image: "/images/hoc2022-hello-world-space.png", url: "https://studio.codeorg/s/hello-world-space-2022/reset" }}
 
-{{ featured_tutorial, title: "NASA's Space Jam", body: "இசை, வானியல் மற்றும் குறியீட்டு முறையைப் பற்றி அறிந்துகொள்வதன் மூலம் உண்மையில் உலுக்கும் சூரிய குடும்பத்தை உருவாக்கவும்!", image: "/images/hoc2022-nasa-space-jam.png", url: "https://chandra.si.edu/sound/code/" }}
+{{ featured_tutorial, title: NASA's Space Jam", body: "இசை, வானியல் மற்றும் குறியீட்டு முறையைப் பற்றி அறிந்துகொள்வதன் மூலம் உண்மையில் உலுக்கும் சூரிய குடும்பத்தை உருவாக்கவும்!", image: /images/hoc2022-nasa-space-jam.png"url: "https://chandra.si.edu/sound/code/" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_explore_activities.ta-IN.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_explore_activities.tr-TR.md.partial [hunk 5]
```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "Merhaba Dünya: Uzay", body: "Karakterler ve \"Merhaba dünyalar!\" diyen seslerle galaksiler arası sahneler yaratın", image: "/images/hoc2022-hello-world-space.png", url: "https://studio.code.org/s/hello-world-space-2022/reset" }}
+{{ featured_tutorial, title: “Hello World: Space”, body: “Karakterler ve seslerle galaksiler arası sahneler yaratın, \"Merhaba dünyalar!\"”, image: "/images/hoc2022-hello-world-space.png“, url: "https://studio.code.org/s/hello-world-space-2022/reset" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_explore_activities.tr-TR.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.ar-SA.md.partial [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
+{{ featured_tutorial, title: "الرياضات"، body: "اصنع لعبة كرة سلة أو امزج وطابق بين الألعاب الرياضية.", image: "/shared/images/courses/logo_tall_sports.jpg", url: "https://code.org/athletes" }}
 
-{{ featured_tutorial, title: "فلابي كود", body: "هل تريد كتابة لعبتك الخاصة في أقل من 10 دقائق؟ جرب برنامجنا التعليمي الخاص ببرمجة فلابي كود!", image: "/shared/images/courses/logo_tall_flappy.jpg", url: "https://studio.code.org/s/flappy/reset" }}
+{{ featured_tutorial, title: "فلابي كود"، body: "هل تريد كتابة لعبتك الخاصة في أقل من 10 دقائق؟ جرب برنامجنا التعليمي الخاص ببرمجة فلابي كود!"، image: "/shared/images/courses/logo_tall_flappy.jpg", url: "https://studio.code.org/s/flappy/reset" }}
 
-{{ featured_tutorial, title: "ماين كرافت في ساعة البرمجة", body: "عادت لعبة ماينكرافت في ساعة البرمجة مع نشاط جديد تمامًا! رحلة عبر ماينكرافت باستخدام الكود.", image: "/images/hoc2022-mc-escape.png", url: "https://aka.ms/HourofCode2022" }}
+{{ featured_tutorial, title: "ماين كرافت في ساعة البرمجة"، body: "عادت لعبة ماينكرافت في ساعة البرمجة مع نشاط جديد تمامًا! رحلة عبر ماينكرافت باستخدام الكود."، image: "/images/hoc2022-mc-escape.png", url: "https://aka.ms/HourofCode2022" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.ar-SA.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.hi-IN.md.partial [hunk 7]
```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "Hello World: Soccer", body: "सॉकर पिच पर इंटरैक्टिव दृश्यों को कोड करते समय आप टीम के कप्तान होते हैं। GOOOAAALLL!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
+{{featured_tutorial, title: “Hello World: Soccer”, body: “सॉकर पिच पर इंटरैक्टिव दृश्यों को कोड करते समय आप टीम के कप्तान होते हैं। GOOOAAALLL!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.hi-IN.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.mr-IN.md.partial [hunk 8]
```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "हॅलो वर्ल्ड: सॉकर", body: "तुम्ही संघाचे कर्णधार आहात कारण तुम्ही सॉकर खेळपट्टीवर परस्परसंवादी दृश्ये कोड करता. गोललल्लल!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
+{{ featured_tutorial, title: "हॅलो वर्ल्ड: सॉकर", body:“तुम्ही संघाचे कर्णधार आहात कारण तुम्ही सॉकर खेळपट्टीवर परस्परसंवादी दृश्ये कोड करता. गोललल्लल!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.mr-IN.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.ro-RO.md.partial [hunk 9]
```diff
@@ -1,6 +1,6 @@
-{{ featured_tutorial, title: "Salutare Lume: Fotbal", body: "Ești căpitanul echipei și codifici scene interative pe terenul de fotfal. GOOOAAALLL!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
+{{ featured_tutorial, title: "Salutare Lume: Fotbal", body: „Ești căpitanul echipei și codifici scene interative pe terenul de fotbal. GOOOAAALLL!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
 
-{{ featured_tutorial, title: "Sport", body: "Crează un joc de basket sau combină dintre diverse sporturi.", image: "/shared/images/courses/logo_tall_sports.jpg", url: "https://code.org/athletes" }}
+{{ featured_tutorial, title: "Sport", body: "Crează un joc de basket sau combină dintre diverse sporturi.", imagine: "/shared/images/courses/logo_tall_sports.jpg", url: "https://code.org/athletes" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.ro-RO.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.ta-IN.md.partial [hunk 10]
```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "ஹலோ வேர்ல்ட்: சாக்கர்", body: "நீங்கள் கால்பந்து ஆடுகளத்தில் ஊடாடும் காட்சிகளைக் குறிப்பதால் நீங்கள் அணியின் கேப்டன். GOOOAALLL!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
+{{ featured_tutorial, title: "ஹலோ வேர்ல்ட்: சாக்கர்", body: "நீங்கள் கால்பந்து ஆடுகளத்தில் ஊடாடும் காட்சிகளைக் குறிப்பதால் நீங்கள் அணியின் கேப்டன். GOOOAALLL!” ,image: "/images/hoc2022-hello-world-soccer.png“, url: "https://studio.code.org/s/hello-world-soccer-2022/reset "}}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.ta-IN.md.partial]: Bad.

File: pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.tr-TR.md.partial [hunk 11]
```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "Merhaba Dünya: Futbol", body: "Futbol sahasında interaktif sahneleri kodlarken takım kaptanı sizsiniz. GOOOOOLL", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
+{{featured_tutorial, title: “Merhaba Dünya: Futbol”, body: “Futbol sahasında interaktif sahneleri kodlarken takım kaptanı sizsiniz. GOOOOOLL", image: "images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.tr-TR.md.partial]: Bad.

**Comment by wilkie** [comment]
all problems manually fixed but need to change them in crowd in

**Pull request merged**